### ISSUE TITLE
[website] Color contrast issue in configuration panel

### ISSUE THREAD

**Issue by chrisdholt** [microsoft/fast#3616]
**Describe the bug; what happened?**
The "FAST Frame" Configuration panel has color contrast issues. Currently the small text/badge at the top of the tab panel which says "FAST FRAME" currently has a contrast ratio of 4.14:1 - a ratio of 4.5:1 is required.

Additionally, the text after it of "DARK MODE" (and others for each panel) also has insufficient color contrast of 4.24 where a contrast ratio of 4.5:1 is required.

**What are the steps to reproduce the issue?**
Go to fast.design, use DevTools in Edge or Chrome and view the tooltip showing insufficient color contrast. You can also sample the colors.

![image](https://user-images.githubusercontent.com/13071055/89234849-c92f6980-d5a1-11ea-87cb-e03e14abc2a6.png)

If this is an issue with the recipes, we should address it with the recipes.

**Comment by chrisdholt** [comment]
@bheston can you take a look at this in light of the work you just did on the panel?

**Comment by bheston** [comment]
The issue is the background color context of the tab panel is darker than the applied styling. This has always been an issue in that you can't override the `background-color` in css for any element that contains other UI or the children will all render as if they are in the parent context.

In the React components we would have typically wrapped the `tab-panel` in a `<Background>` and set the desired background color using a recipe there. That would cascade and all the children of the panel would render correctly. I believe we planned to address this, but I wasn't sure the correct way to fix it. I wrapped the entire `fast-tabs` in a new design system and hard-coded the desired background color and the text color updated as expected. @nicholasrice or @EisenbergEffect for support on the best way to reset the --background-color for the panels only and not the entire tab set using a recipe instead of a fixed value.

**Comment by chrisdholt** [comment]
@bheston can you push your branch? A new design system provider context within tabs should suffice here but when I tested it did not work as expected. Considering it's nested within the fast-tabs control, that should take precedence.

**Comment by chrisdholt** [comment]
Closing as I haven't repro'd this recently.
